pkg/repository/v1manifest/types.go

```diff
@@ -116,6 +116,8 @@ type Component struct {
 
 // ComponentItem object
 type ComponentItem struct {
+	Name      string `json:"name"`
+	Desc      string `json:"description"`
```

Comment:
I think it is better not to add these here since they can be changed when a component is updated and then we would need to touch index.json.

Reply:
Yes. But it introduces another problem: the tiup list must fetch all componet.json to display component discription

Reply:
Hmmm, true. I think this is OK for now, but long term maybe we can move them to snapshot.json.

Reply:
OK